src/callbacks_step/analysis.jl

```diff
@@ -309,6 +316,22 @@ function (analysis_callback::AnalysisCallback)(io, du, u, u_ode, t, semi)
     end
   end
 
+
+  # Entropy conservation errror
+  if :entropy_conservation_error in analysis_errors
+    @unpack initial_entropy_state_integrals = analysis_callback
+    entropy_state_integrals = integrate(entropy, u_ode, semi)
+
+    if mpi_isroot()
+      print(" |(∑S₀ - ∑S)| / | ∑S₀ | : ")
+        err = (abs(initial_entropy_state_integrals - entropy_state_integrals) 
+                / abs(initial_entropy_state_integrals))
+        @printf("  % 10.8e", err)
+        @printf(io, "  % 10.8e", err)
+      println()
+    end
+  end
+
```

Comment:
I left this to discuss whether to keep this additional analysis function or to remove it, since the change in entropy is already monitored by the residual.

Reply:
I believe this is different from what we currently measure. IIRC, the current EC check only verifies that the time derivative does not change entropy. What you are looking at here is the integral entropy over the entire simulation, including time integration.

Reply:
Yes, the question was more along the line of do we want to keep this additional way of monitoring it.

Reply:
Let's keep it for now in here and see what shakes out when doing the actual review. Would that be ok for you? Or does it cause additional overhead for you if we defer the decision to a later point?